src/Land/Model/SoilHeatParameterizations.jl

```diff
@@ -0,0 +1,196 @@
+"""
+    SoilHeatParameterizations
+
+Functions for volumetric heat capacity, internal energy as function of temperature,
```

Comment:
same here, volumetric internal energy

Reply:
maybe missing some functions in this description

Reply:
fixed!